teams/models/team.py

```diff
@@ -4,9 +4,11 @@
 
 class Team(models.Model):
 	team_name = models.CharField(max_length = 40)
+	abbr_name = models.CharField(max_length = 5, default = None)
 	division_ind = models.CharField(max_length = 1, default = 'E',choices=DIVISIONS)
-	is_active = models.BooleanField(default=False)
-	created_on = models.DateTimeField(auto_now_add=True)
+	
+	is_active = models.BooleanField(default=False) #needed?
+	date_founded = models.DateTimeField(auto_now_add=True)
```

Comment:
Not huge fan of date founded 🙃 I think it is more consistent or universally applicable to use `created_on`